lib/testers/rule-tester.js

```diff
@@ -429,7 +429,7 @@ RuleTester.prototype = {
                     item.errors.length, item.errors.length === 1 ? "" : "s", messages.length, util.inspect(messages)));
 
                 for (let i = 0, l = item.errors.length; i < l; i++) {
-                    assert.ok(!("fatal" in messages[i]), `A fatal parsing error occurred: ${messages[i].message}`);
+                    assert.ok(!messages[i].fatal, `A fatal parsing error occurred: ${messages[i].message}`);
```

Comment:
At least one reason I can think of is that a formatter like the JSON formatter would output fewer bytes. That formatter has been known to crash with extremely large output, due to the fact that the entire JSON string has to be buffered in memory.